eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/actions/DirectionsLabelFactory.java

```diff
@@ -44,6 +44,6 @@ public DirectionsLabelFactory(Iterable<Location> locations)
     @Override
     public CharSequence create(Context context)
     {
-        return new AddressName(mLocations).value(context.getString(R.string.schedjoules_action_directions));
+        return new AddressName(mLocations).value(context.getString(R.string.schedjoules_action_directions).toUpperCase());
```

Comment:
The Directions action can have 2 labels now: DIRECTIONS in case we have no address and the address in normal case if we have it, although I am not sure if the first case would ever happen.
What would the BorderlessButton style add? Can we simply add the icon there? Now the whole layout, the icon and the text acts as a button when pressed.
In any case, can we address it in a separate story if needed?